### ISSUE TITLE
installation error

### ISSUE THREAD

**Issue by sivajik34** [magento/magento2#892]
Starting Magento installation:
File permissions check...

Enabling Maintenance Mode...

Installing deployment configuration...

Installing database schema:
Schema creation/updates:
Module 'Magento_Core':

Module 'Magento_Store':

Module 'Magento_Directory':

Module 'Magento_Backup':

Module 'Magento_Theme':

Module 'Magento_Eav':

Module 'Magento_Customer':

Module 'Magento_CatalogImportExport':

Module 'Magento_Indexer':

Module 'Magento_Cms':

Module 'Magento_Backend':

Module 'Magento_Catalog':

Module 'Magento_Rule':

Module 'Magento_CatalogInventory':

Module 'Magento_Payment':

Module 'Magento_CheckoutAgreements':
Include /var/www/html/magento2/app/code/Magento/CheckoutAgreements/sql/checkoutagreements_setup/install-2.0.0.php
[ERROR] exception 'PDOException' with message 'SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'checkout_agreement' already exists' in /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Statement/Pdo.php:228

Stack trace:

#0 /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Statement/Pdo.php(228): PDOStatement->execute(Array)

#1 /var/www/html/magento2/lib/internal/Magento/Framework/DB/Statement/Pdo/Mysql.php(91): Zend_Db_Statement_Pdo->_execute(Array)

#2 /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Statement.php(303): Magento\Framework\DB\Statement\Pdo\Mysql->_execute(Array)

#3 /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Abstract.php(480): Zend_Db_Statement->execute(Array)

#4 /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php(238): Zend_Db_Adapter_Abstract->query('CREATE TABLE `c...', Array)

#5 /var/www/html/magento2/lib/internal/Magento/Framework/DB/Adapter/Pdo/Mysql.php(416): Zend_Db_Adapter_Pdo_Abstract->query('CREATE TABLE `c...', Array)

#6 /var/www/html/magento2/lib/internal/Magento/Framework/DB/Adapter/Pdo/Mysql.php(1952): Magento\Framework\DB\Adapter\Pdo\Mysql->query('CREATE TABLE `c...')

#7 /var/www/html/magento2/app/code/Magento/CheckoutAgreements/sql/checkoutagreements_setup/install-2.0.0.php(60): Magento\Framework\DB\Adapter\Pdo\Mysql->createTable(Object(Magento\Framework\DB\Ddl\Table))

#8 /var/www/html/magento2/setup/module/Magento/Setup/src/Module/SetupModule.php(271): include('/var/www/html/m...')

#9 /var/www/html/magento2/setup/module/Magento/Setup/src/Module/SetupModule.php(183): Magento\Setup\Module\SetupModule->includeFile('/var/www/html/m...')

#10 /var/www/html/magento2/setup/module/Magento/Setup/src/Module/SetupModule.php(153): Magento\Setup\Module\SetupModule->applySchemaUpdates('install', '', '2.0.0')

#11 /var/www/html/magento2/setup/module/Magento/Setup/src/Model/Installer.php(559): Magento\Setup\Module\SetupModule->applyUpdates()

#12 [internal function]: Magento\Setup\Model\Installer->installSchema()

#13 /var/www/html/magento2/setup/module/Magento/Setup/src/Model/Installer.php(296): call_user_func_array(Array, Array)

#14 /var/www/html/magento2/setup/module/Magento/Setup/src/Controller/Install.php(79): Magento\Setup\Model\Installer->install(Array)

#15 /var/www/html/magento2/vendor/zendframework/zend-mvc/Zend/Mvc/Controller/AbstractActionController.php(83): Magento\Setup\Controller\Install->startAction()

#16 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))

#17 /var/www/html/magento2/vendor/zendframework/zend-eventmanager/Zend/EventManager/EventManager.php(468): call_user_func(Array, Object(Zend\Mvc\MvcEvent))

#18 /var/www/html/magento2/vendor/zendframework/zend-eventmanager/Zend/EventManager/EventManager.php(207): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))

#19 /var/www/html/magento2/vendor/zendframework/zend-mvc/Zend/Mvc/Controller/AbstractController.php(117): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))

#20 /var/www/html/magento2/vendor/zendframework/zend-mvc/Zend/Mvc/DispatchListener.php(114): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))

#21 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))

#22 /var/www/html/magento2/vendor/zendframework/zend-eventmanager/Zend/EventManager/EventManager.php(468): call_user_func(Array, Object(Zend\Mvc\MvcEvent))

#23 /var/www/html/magento2/vendor/zendframework/zend-eventmanager/Zend/EventManager/EventManager.php(207): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))

#24 /var/www/html/magento2/vendor/zendframework/zend-mvc/Zend/Mvc/Application.php(313): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))

#25 /var/www/html/magento2/setup/index.php(24): Zend\Mvc\Application->run()

#26 {main}

Next exception 'Zend_Db_Statement_Exception' with message 'SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'checkout_agreement' already exists, query was: CREATE TABLE `checkout_agreement` (

  `agreement_id` int UNSIGNED NOT NULL auto_increment COMMENT 'Agreement Id' ,

  `name` varchar(255) NULL COMMENT 'Name' ,

  `content` text NULL COMMENT 'Content' ,

  `content_height` varchar(25) NULL COMMENT 'Content Height' ,

  `checkbox_text` text NULL COMMENT 'Checkbox Text' ,

  `is_active` smallint NOT NULL default '0' COMMENT 'Is Active' ,

  `is_html` smallint NOT NULL default '0' COMMENT 'Is Html' ,

  PRIMARY KEY (`agreement_id`)

) COMMENT='Checkout Agreement' ENGINE=INNODB charset=utf8 COLLATE=utf8_general_ci' in /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Statement/Pdo.php:235

Stack trace:

#0 /var/www/html/magento2/lib/internal/Magento/Framework/DB/Statement/Pdo/Mysql.php(91): Zend_Db_Statement_Pdo->_execute(Array)

#1 /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Statement.php(303): Magento\Framework\DB\Statement\Pdo\Mysql->_execute(Array)

#2 /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Abstract.php(480): Zend_Db_Statement->execute(Array)

#3 /var/www/html/magento2/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php(238): Zend_Db_Adapter_Abstract->query('CREATE TABLE `c...', Array)

#4 /var/www/html/magento2/lib/internal/Magento/Framework/DB/Adapter/Pdo/Mysql.php(416): Zend_Db_Adapter_Pdo_Abstract->query('CREATE TABLE `c...', Array)

#5 /var/www/html/magento2/lib/internal/Magento/Framework/DB/Adapter/Pdo/Mysql.php(1952): Magento\Framework\DB\Adapter\Pdo\Mysql->query('CREATE TABLE `c...')

#6 /var/www/html/magento2/app/code/Magento/CheckoutAgreements/sql/checkoutagreements_setup/install-2.0.0.php(60): Magento\Framework\DB\Adapter\Pdo\Mysql->createTable(Object(Magento\Framework\DB\Ddl\Table))

#7 /var/www/html/magento2/setup/module/Magento/Setup/src/Module/SetupModule.php(271): include('/var/www/html/m...')

#8 /var/www/html/magento2/setup/module/Magento/Setup/src/Module/SetupModule.php(183): Magento\Setup\Module\SetupModule->includeFile('/var/www/html/m...')

#9 /var/www/html/magento2/setup/module/Magento/Setup/src/Module/SetupModule.php(153): Magento\Setup\Module\SetupModule->applySchemaUpdates('install', '', '2.0.0')

#10 /var/www/html/magento2/setup/module/Magento/Setup/src/Model/Installer.php(559): Magento\Setup\Module\SetupModule->applyUpdates()

#11 [internal function]: Magento\Setup\Model\Installer->installSchema()

#12 /var/www/html/magento2/setup/module/Magento/Setup/src/Model/Installer.php(296): call_user_func_array(Array, Array)

#13 /var/www/html/magento2/setup/module/Magento/Setup/src/Controller/Install.php(79): Magento\Setup\Model\Installer->install(Array)

#14 /var/www/html/magento2/vendor/zendframework/zend-mvc/Zend/Mvc/Controller/AbstractActionController.php(83): Magento\Setup\Controller\Install->startAction()

#15 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))

#16 /var/www/html/magento2/vendor/zendframework/zend-eventmanager/Zend/EventManager/EventManager.php(468): call_user_func(Array, Object(Zend\Mvc\MvcEvent))

#17 /var/www/html/magento2/vendor/zendframework/zend-eventmanager/Zend/EventManager/EventManager.php(207): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))

#18 /var/www/html/magento2/vendor/zendframework/zend-mvc/Zend/Mvc/Controller/AbstractController.php(117): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))

#19 /var/www/html/magento2/vendor/zendframework/zend-mvc/Zend/Mvc/DispatchListener.php(114): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))

#20 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))

#21 /var/www/html/magento2/vendor/zendframework/zend-eventmanager/Zend/EventManager/EventManager.php(468): call_user_func(Array, Object(Zend\Mvc\MvcEvent))

#22 /var/www/html/magento2/vendor/zendframework/zend-eventmanager/Zend/EventManager/EventManager.php(207): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))

#23 /var/www/html/magento2/vendor/zendframework/zend-mvc/Zend/Mvc/Application.php(313): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))

#24 /var/www/html/magento2/setup/index.php(24): Zend\Mvc\Application->run()

#25 {main}

Next exception 'Exception' with message 'Error in file: "/var/www/html/magento2/app/code/Magento/CheckoutAgreements/sql/checkoutagreements_setup/install-2.0.0.php" - SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'checkout_agreement' already exists, query was: CREATE TABLE `checkout_agreement` (

  `agreement_id` int UNSIGNED NOT NULL auto_increment COMMENT 'Agreement Id' ,

  `name` varchar(255) NULL COMMENT 'Name' ,

  `content` text NULL COMMENT 'Content' ,

  `content_height` varchar(25) NULL COMMENT 'Content Height' ,

  `checkbox_text` text NULL COMMENT 'Checkbox Text' ,

  `is_active` smallint NOT NULL default '0' COMMENT 'Is Active' ,

  `is_html` smallint NOT NULL default '0' COMMENT 'Is Html' ,

  PRIMARY KEY (`agreement_id`)

) COMMENT='Checkout Agreement' ENGINE=INNODB charset=utf8 COLLATE=utf8_general_ci' in /var/www/html/magento2/setup/module/Magento/Setup/src/Module/SetupModule.php:197

Stack trace:

#0 /var/www/html/magento2/setup/module/Magento/Setup/src/Module/SetupModule.php(153): Magento\Setup\Module\SetupModule->applySchemaUpdates('install', '', '2.0.0')

#1 /var/www/html/magento2/setup/module/Magento/Setup/src/Model/Installer.php(559): Magento\Setup\Module\SetupModule->applyUpdates()

#2 [internal function]: Magento\Setup\Model\Installer->installSchema()

#3 /var/www/html/magento2/setup/module/Magento/Setup/src/Model/Installer.php(296): call_user_func_array(Array, Array)

#4 /var/www/html/magento2/setup/module/Magento/Setup/src/Controller/Install.php(79): Magento\Setup\Model\Installer->install(Array)

#5 /var/www/html/magento2/vendor/zendframework/zend-mvc/Zend/Mvc/Controller/AbstractActionController.php(83): Magento\Setup\Controller\Install->startAction()

#6 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))

#7 /var/www/html/magento2/vendor/zendframework/zend-eventmanager/Zend/EventManager/EventManager.php(468): call_user_func(Array, Object(Zend\Mvc\MvcEvent))

#8 /var/www/html/magento2/vendor/zendframework/zend-eventmanager/Zend/EventManager/EventManager.php(207): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))

#9 /var/www/html/magento2/vendor/zendframework/zend-mvc/Zend/Mvc/Controller/AbstractController.php(117): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))

#10 /var/www/html/magento2/vendor/zendframework/zend-mvc/Zend/Mvc/DispatchListener.php(114): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))

#11 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))

#12 /var/www/html/magento2/vendor/zendframework/zend-eventmanager/Zend/EventManager/EventManager.php(468): call_user_func(Array, Object(Zend\Mvc\MvcEvent))

#13 /var/www/html/magento2/vendor/zendframework/zend-eventmanager/Zend/EventManager/EventManager.php(207): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))

#14 /var/www/html/magento2/vendor/zendframework/zend-mvc/Zend/Mvc/Application.php(313): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))

#15 /var/www/html/magento2/setup/index.php(24): Zend\Mvc\Application->run()

#16 {main}

**Comment by jameswithers** [comment]
@sivajik34  Clue is in the error message: `Table 'checkout_agreement' already exists'`. Have you ensured the schema is empty and tried again?

**Comment by sivajik34** [comment]
Yes Database is empty.its new one.

**Comment by Galch24** [comment]
I tried to install Magento with composer about 8 times. And every time I started from the beginning with clear database. 
Firstly I installed it on MAMP web-server, then I repeat installing on cleared new web-server with Apache, PHP 5.5 and new (just from the box) MySQL 5.6. So, yes. I'm pretty sure that my scheme was empty.

**Comment by Galch24** [comment]
I tried install with Sample data and without it. The same errors comes out.

My system:
Mac OS 10.7 Lion.

$ httpd -v
Server version: Apache/2.2.26 (Unix)
Server built: Dec 15 2013 18:33:29

$ mysql -u root -p
Server version: 5.6.2-m5 MySQL Community Server (GPL)

$ php -v
PHP 5.5.27 (cli) (built: Jul 10 2015 23:02:22)
Copyright (c) 1997-2015 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2015 Zend Technologies
with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2015, by Zend Technologies
with Xdebug v2.2.5, Copyright (c) 2002-2014, by Derick Rethans

I uses clear MySQL, PHP, Apache installed with this manual http://coolestguidesontheplanet.com/how-to-install-php-mysql-apache-on-os-x-10-6/

So, I have errors that seems to be because of low version of MySQL. But I already updated MySQL and PHP to max possible for Mac OS 10.7 Lion.

Firstly I was tried install Magento on MAMP 3.0.7 with php 5.6, mySql 5.6. But it caused the same errors with the same step (module). So, I decided repeat all steps on cleared MySQL, PHP, Apache. However, this has no effect

What else I can do to recognize whats wrong with my installation of Magento?
